twatbot.py

```diff
@@ -485,14 +490,18 @@ def httpsrv_client_thread(c, evt_done):
 	elif req['url'].startswith('/index.html'):
 		vars={}
 		vars['page'] = 0
+		user=[]
```

Comment:
i think that stuff here is no longer needed, right ? we replaced it with the js...